danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/DangerDSL.kt

```diff
@@ -29,4 +29,6 @@ data class DangerDSL(
                 get() = _bitBucketServer != null
         val onGitLab
             get() = _gitlab != null
+
+        val utils = Utils()
```

Comment:
what about provide a set of extensions functions instead? then we don't need to put an additional object for that.
Same way as implemented for the `onGitHub` and `onBitbucket`

Reply:
We use utils on the other danger, I prefer to keep the consistency, also because the functions are the same